src-plugins/itkFilters/itkFiltersBinaryCloseProcess_p.h

```diff
@@ -72,8 +72,8 @@ class itkFiltersBinaryCloseProcessPrivate : public itkMorphologicalFiltersProces
         QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
 
         if (isRadiusInPixels)
-            newSeriesDescription += " Bin.Close filter\n("+ QString::number(radius[0])+", "+
-            QString::number(radius[1])+", "+ QString::number(radius[2])+" pixels)";
+            newSeriesDescription += " Bin.Close filter\n("+ QString::number(floor(radius[0]))+", "+
+            QString::number(floor(radius[1]))+", "+ QString::number(floor(radius[2]))+" pixels)";
```

Comment:
:D, indeed. So, we can do as Maxime suggested: allow the user to enter only an integer, or do the "floor(val+0.5)", or let like that. I don't know, as you want ^^. Personally i would put the 0.5 because it's easy, and accurate ^^